src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
```

Comment:
These cases predate our efforts to be very specific about browser bugs in `// Supports` comments, but they reinforce that being specific can save us a lot of work down the road.

I agree it is difficult and pointless to go back and see whether Chrome 18, for instance, has this bug. We're not planning to take any of this out for a while, so a labor-intensive investigation to get an accurate version may not be worth the effort until that point.

Still, there is a [growing problem with Chrome fragmentation](http://www.quirksmode.org/blog/archives/2015/02/chrome_continue.html) and big phone vendors like Samsung are stuck on Chrome 28. In the general case we can't just test on desktop Chrome 28 but infer that the problem also exists on Samsung's Chrome 28.